src/lib/snyk-to-html.ts

```diff
@@ -83,9 +85,35 @@ function metadataForVuln(vuln: any) {
     description: vuln.description || 'No description available.',
     fixedIn: vuln.fixedIn,
     packageManager: vuln.packageManager,
+    version: vuln.version,
+    cvssScore: vuln.cvssScore,
+    cveSpaced: cveSpaced || 'No CVE found.',
+    cveLineBreaks: cveLineBreaks || 'No CVE found.',
+    disclosureTime: dateFromDateTimeString(vuln.disclosureTime || ''),
+    publicationTime: dateFromDateTimeString(vuln.publicationTime || ''),
+    license: vuln.license || undefined
   };
 }
 
+function concatenateCVEs(vuln: any) {
+  let cveSpaced = ''
+  let cveLineBreaks = ''
+
+  if (!(typeof vuln.identifiers === "undefined")) {
```

Comment:
```suggestion
  if (vuln.identifiers) {
```

Couldn't you check only for its existence?

Reply:
you're right, updated.